app/javascript/dashboard/routes/dashboard/helpcenter/helpcenter.routes.js

```diff
@@ -31,7 +31,7 @@ const portalRoutes = [
     component: PortalsArticlesIndexPage,
   },
   {
-    path: getPortalRoute(':portalSlug/:locale/articles/new'),
+    path: getPortalRoute(':portalSlug/:locale/:categorySlug?/articles/new'),
```

Comment:
Why do we need this?

Reply:
@pranavrajs We are using this `categorySlug` param to automatically select the category when creating a new article if a filter is applied in article list.